main.go

```diff
@@ -40,25 +42,47 @@ func main() {
 	o.Path("/login-complete").Methods("GET").HandlerFunc(handler.LoginComplete)
 
 	a := r.PathPrefix("/api").Subrouter()
-	a.Use(authenticator.Middleware)
+	// a.Use(authenticator.Middleware)
 
 	s := a.PathPrefix("/elections").Subrouter()
-	s.Methods("GET").HandlerFunc(handler.GetElectionIds)
-	s.Methods("POST").HandlerFunc(handler.CreateElection)
+	s.Methods("GET").HandlerFunc(handler.GetElections)
+	s.Path("/create").Methods("POST").HandlerFunc(handler.CreateElection)
 
 	s = a.PathPrefix("/election/{electionId}").Subrouter()
+	s.Methods("GET").HandlerFunc(handler.GetElectionInfo)
+	s.Methods("POST").HandlerFunc(handler.ModifyElection)
 	s.Path("/publish").Methods("PUT").HandlerFunc(handler.PublishElection)
 	s.Path("/close").Methods("PUT").HandlerFunc(handler.CloseElection)
 	s.Path("/vote").Methods("POST").HandlerFunc(handler.CastVote)
-	s.Path("/votes").Methods("GET").HandlerFunc(handler.GetVotes)
-	s.Path("/voters").Methods("GET").HandlerFunc(handler.GetEligibleVoters)
-	s.Path("/voters").Methods("PUT").HandlerFunc(handler.AddEligibleVoters)
+	s.Path("/votes").Methods("GET").HandlerFunc(handler.GetElectionVotes)
+	s.Path("/votes/count").Methods("GET").HandlerFunc(handler.CountElectionVotes)
+
+	s = a.PathPrefix("/candidates").Subrouter()
+	s.Methods("GET").HandlerFunc(handler.GetAllCandidates)
+	s.Path("/create").HandlerFunc(handler.CreateCandidate)
```

Comment:
Don't you need to put a `.Methods("POST")` here? Or would all methods be handled `CreateCandidate` otherwise?

Also, same thing as with `/elections` sounds a bit strange to me with `POST /candidates/create`.

Reply:
Yeah, there should be a post

I think this makes sense, will change this